src/assets/scripts/client/App.js

```diff
@@ -199,7 +199,7 @@ export default class App {
         this.uiController = new UiController(this.$element, this.gameController, this.airportController);
         this.canvasController = new CanvasController(this.$element, this.navigationLibrary, this.gameController);
         this.tutorialView = new TutorialView(this.$element);
-        this.aircraftCommander = new AircraftCommander(this.airportController, this.navigationLibrary, this.gameController, this.uiController);
+        this.aircraftCommander = new AircraftCommander(this.airportController, this.navigationLibrary, this.gameController, this.uiController, this.aircraftController.onRequestToChangeTransponderCode);
```

Comment:
I really don't like this, but its a necessary evil at the moment.  Once #72 is in place, this win't look so messy.